retail/interactive-tutorials/src/main/java/search/SearchSimpleQuery.java

```diff
@@ -53,7 +55,9 @@ public static void searchResponse(String defaultSearchPlacementName) throws IOEx
             .setPageSize(pageSize)
             .build();
     System.out.println("Search request: " + searchRequest);
-
+    
+// [END retail_search_simple_query]
+    
```

Comment:
Lint check is failing due to indentation. 
```suggestion
    // [END retail_search_simple_query]
```